### PR TITLE
hwdef:correct JFB110 defaults

### PR DESCRIPTION
**Pull request by jfbblue0922**
Serial 5 of JFB110 has only input and is designed for RCIN. So we would like to set default parameters for serial 5.

### CODE REVIEW

**Comment by rmackay9** [comment]
@jfbblue0922,

Thanks for this.  So @Hwurzburg's [PR from two weeks ago removed this](https://github.com/ArduPilot/ardupilot/pull/25503).  Sorry about that, I should have gotten your approval before we merged it.

I guess you've checked this change and you're sure that these lines are required in order for RC input to work?

**Comment by jfbblue0922** [comment]
@rmackay9
Thank you for the reply.

We have verified that this feature works.

Most of our customers use S.BUS.
This SERIAL5 with input only is hard to  use for other functions, we would like to improve its convenience.
The same goes for SERIAL6 with output only.

Thank you.

**Comment by Hwurzburg** [comment]
@jfbblue0922 SBUS should work fine on the PPM input since its timer interrupt based,as will any unidirectional RC protocol....designating SERIAL5 RX as RCIN does not really enhance the user experience since any RC protocol that it can detect, PPM input will detect. 
Normally, the UART RCin protocol is used with a full UART(RX and TX) for the bi-directional protocols like CRSF/ELRS,SRXL2,etc...personally, I think defaulting that pin to RC will be a little confusing...Since only its RX is pinned out, it will more likely find use as ESC telem in, or RPM,etc.
Its your autopilot, and if you wish to set its default to RCIN protocol is your choice we will, of course. It will indeed function that way (although it loses the PPM ability) but it's not needed to support SBUS (as I have noted in its documentation on the wiki), and might be a little confusing(although I can update the wiki doc to explain that it could be used as a second RC receiver input for the unidirectional protocols except PPM..although you could do that on any UART)

**Comment by muramura** [comment]
If you don't have a UART dedicated to RCIN, it would be better to describe in the spec that you don't have RCIN and let the user set up UART5.
UART5 is a general-purpose UART.
It is up to the user to decide how to use it.

**Comment by Hwurzburg** [comment]
setting this has no impact on on the PPM pins functionality....since you insist on this, I will change the documentation when it is merged..

File: libraries/AP_HAL_ChibiOS/hwdef/JFB110/defaults.parm
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/JFB110/defaults.parm:5]: not needed,remove

File: libraries/AP_HAL_ChibiOS/hwdef/JFB110/defaults.parm
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/JFB110/defaults.parm:7]: not needed, remove

**Review by Hwurzburg — DISMISSED** [review]
those are not needed for SBUS and will make other rc systems fail on this pin

File: libraries/AP_HAL_ChibiOS/hwdef/JFB110/defaults.parm
Comment (rmackay9) [reply libraries/AP_HAL_ChibiOS/hwdef/JFB110/defaults.parm]: removed, txs

File: libraries/AP_HAL_ChibiOS/hwdef/JFB110/defaults.parm
Comment (rmackay9) [reply libraries/AP_HAL_ChibiOS/hwdef/JFB110/defaults.parm]: remove, txs

**Review by rmackay9 — APPROVED** [review]

**Pull request merged**
